### PR TITLE
Fixed resource file not being set when making localized calls in Grad…

### PR DESCRIPTION
**Pull request by HSAR**
…le build.

### CODE REVIEW

File: Tasks/Gradle/gradlesonar.ts [hunk 1]
```diff
@@ -11,6 +11,9 @@ import trm = require('vsts-task-lib/toolrunner');
 import sqCommon = require('sonarqube-common/sonarqube-common');
 import {SonarQubeEndpoint} from 'sonarqube-common/sonarqube-common';
 
+// Set up localization resource file
+tl.setResourcePath(path.join( __dirname, 'task.json'));
```
Comment (bgavrilMS) [inline Tasks/Gradle/gradlesonar.ts]: Is this necessary ? Isn't it enough to set it once in the GradleTask.ts ?

**Comment by bgavrilMS** [comment]
LGTM

File: Tasks/Gradle/gradlesonar.ts (hunk 1 above)
Comment (HSAR) [inline Tasks/Gradle/gradlesonar.ts]: Discovered in testing that in fact no, it's not.

Comment (bgavrilMS) [inline Tasks/Gradle/gradlesonar.ts]: Good to know!

**Pull request merged**